pyhap/accessory.py

```diff
@@ -4,9 +4,7 @@
 import struct
 import threading
 
-import base36
 import ed25519
-from pyqrcode import QRCode
```

Comment:
Small request, if it's not too much: Can you do it like a feature switch:
```python
try:
   from pyqrcode import QRCode
   HAS_QR = True
except ImportError:
   HAS_QR = False
...
...
if not HAS_QR:
   print("Enter the below code...")
print(The QR code)
```
I think this will make the code easier to read and maintain, because you still keep the imports in one place and the logic in another.

As an optimisation, I think this can even be done in the __init__py:
```python
#__init__.py

try:
   import pyqrcode
   HAS_QR = True
except ImportError:
   HAS_QR = False

try:
   import X
   HAS_X = True
except ImportError:
   HAS_X = False

# foo.py
from pyhap import HAS_QR
if HAS_QR:
   import pyqrcode

# bar.py
from pyhap import HAS_X
if HAS_X:
   import X
```